### PR TITLE
docs: add configure connector (en-US)

### PR DESCRIPTION
**Pull request by darcyYe**

### CODE REVIEW

File: docs/docs/recipes/configure-connectors/README.md [hunk 1]
```diff
@@ -3,3 +3,26 @@ sidebar_position: 5
...
+_Connectors_ are the bridge between Logto and other third-party vendors who provide short message service (SMS), email service, or user information on wildly accepted social media.
+
+Logto Team provides you with many connectors to give end-users access to third-party providers' services. Such as Google, GitHub, Facebook, SendGrid mail, etc.
+
+You may feel annoyed when using such robust connectors:sob:?<br/>
+Don't worry, this doc will show you an easy way to have things done!
+
+## :watch: Kick-off :zap:
+
+Let's start from [Admin Console](https://logto.dev/console), signing in with your Logto account ("admin" access needed). You may [create an admin account](../../tutorials/get-started/#create-an-admin-account) if you don't have one.
```
Comment (gao-sun) [inline docs/docs/recipes/configure-connectors/README.md:16]: should use their own env
```suggestion
Let's start from Admin Console, signing in with your Logto account ("admin" access needed). You may [create an admin account](../../tutorials/get-started.mdx#create-an-admin-account) if you don't have one.
```

File: docs/docs/recipes/configure-connectors/README.md [hunk 2]
```diff
@@ -3,3 +3,26 @@ sidebar_position: 5
...
+
+Click the **Connectors** button on the sidebar :neutral_face:
+
+:ghost:**Welcome to the world of _connectors_**:ghost:
```
Comment (gao-sun) [inline docs/docs/recipes/configure-connectors/README.md:20]:
```suggestion
:ghost: **Welcome to the world of _connectors_** :ghost:
```

File: docs/docs/recipes/configure-connectors/README.md [hunk 3]
```diff
@@ -3,3 +3,26 @@ sidebar_position: 5
 ---
 
 # 🪛 Configure connectors
+
+_Connectors_ are the bridge between Logto and other third-party vendors who provide short message service (SMS), email service, or user information on wildly accepted social media.
+
+Logto Team provides you with many connectors to give end-users access to third-party providers' services. Such as Google, GitHub, Facebook, SendGrid mail, etc.
+
+You may feel annoyed when using such robust connectors:sob:?<br/>
```
[the remark this answers is not in the document]
Comment (darcyYe) [reply docs/docs/recipes/configure-connectors/README.md]: Tried to build a comparison between the abilities of connectors and the loss of developers when first see this conception.
Rephrased in another way.

File: docs/docs/recipes/configure-connectors/README.md [hunk 4]
```diff
@@ -3,3 +3,27 @@ sidebar_position: 5
...
 
 # 🪛 Configure connectors
+
+_Connectors_ are the bridge between Logto and other third-party vendors who provide short message service (SMS), email service, or user information on wildly accepted social media.
+
+Logto provides you with many connectors to give end-users access to third-party providers. Such as Google, GitHub, Facebook, SendGrid mail, etc.
+
+You may feel lost when using connectors as this is a brand new concept :sob:
+
+Don't worry; this doc will show you an easy way to have things done!
```
Comment (gao-sun) [inline docs/docs/recipes/configure-connectors/README.md:13]:
```suggestion
Don't worry. This doc will show you an easy way to have things done!
```

File: docs/docs/recipes/configure-connectors/README.md [hunk 5]
```diff
@@ -3,3 +3,27 @@ sidebar_position: 5
 ---
 
 # 🪛 Configure connectors
+
+_Connectors_ are the bridge between Logto and other third-party vendors who provide short message service (SMS), email service, or user information on wildly accepted social media.
+
+Logto provides you with many connectors to give end-users access to third-party providers. Such as Google, GitHub, Facebook, SendGrid mail, etc.
+
+You may feel lost when using connectors as this is a brand new concept :sob:
```
Comment (gao-sun) [inline docs/docs/recipes/configure-connectors/README.md:11]:
```suggestion
You may feel lost when using connectors as this is a brand new concept. :sob:
```

File: docs/docs/recipes/configure-connectors/README.md [hunk 6]
```diff
@@ -3,3 +3,27 @@ sidebar_position: 5
...
+
+Don't worry; this doc will show you an easy way to have things done!
+
+## :watch: Kick-off :zap:
+
+Let's start from Admin Console, signing in with your Logto account ("admin" access needed). You may [create an admin account](../../tutorials/get-started/README.mdx#create-an-admin-account) if you don't have one.
```
Comment (gao-sun) [inline docs/docs/recipes/configure-connectors/README.md:17]:
```suggestion
Let's start from Admin Console, signing in with your Logto account ("admin" role needed). You may [create an admin account](../../tutorials/get-started/README.mdx#create-an-admin-account) if you don't have one.
```

File: docs/docs/recipes/configure-connectors/sms-or-email-connector.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+Follow the README on the left and compose the connector config JSON in the text input box on your right. You can check whether the config can work before saving changes.
+
+You can change the current SMS or Email connector in the top-right corner on the connector details page, where a review of the connector README is also available.
+
+:::note
+You can bring **only one _SMS Connector_ (or _Email Connector_)** "alive".
+
+If you change the "living" SMS or Email connector and save changes, the previous SMS or Email connector will be disabled automatically. Also, the config of past connectors will lose track.<br/>
+You may keep the old connectors' config properly if you want to switch back to those connectors in the future.
+:::
```
Comment (gao-sun) [inline docs/docs/recipes/configure-connectors/sms-or-email-connector.md:19]: maybe refer to "Enable SMS or email passcode sign-in" in the end?

File: docs/docs/recipes/configure-connectors/social-connector.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+Suppose you are now on the "Connectors" page in Admin Console. Go to the "Social connectors" tab and add a new connector to enter the configuration process.
+
+:::tip
+You may find that no secondary choice of **_platform_** is required for some connectors such as Google, Facebook, and GitHub. These connectors' **_platform_** are always **_Universal_**. Explore **_platform_** description (TODO: add a link here after connector reference merged) to know more.
+
+Choose a proper **_platform_** from either **_Web_** or **_Native_** for connectors to fit your use case.<br/>
+You can set up a single **_Native_** connector w/o adding a **_Web_** connector if you only provide mobile apps and vice versa.
+:::
+
+Follow the README to compose the connector config JSON with little effort.
```
Comment (gao-sun) [inline docs/docs/recipes/configure-connectors/social-connector.md:17]: ditto

File: docs/docs/recipes/configure-connectors/sms-or-email-connector.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+# SMS or Email Connector
+
+Suppose you are now on the "Connectors" page in Admin Console. Click the _Set up_ button and select an SMS or Email connector to enter the configuration process.
+
+Follow the README on the left and compose the connector config JSON in the text input box on your right. You can check whether the config can work before saving changes.
+
+You can change the current SMS or Email connector in the top-right corner on the connector details page, where a review of the connector README is also available.
+
+:::note
+You can bring **only one _SMS Connector_ (or _Email Connector_)** "alive".
```
Comment (gao-sun) [inline docs/docs/recipes/configure-connectors/sms-or-email-connector.md:15]:
```suggestion
You can bring ONLY ONE _SMS Connector_ (or _Email Connector_) "alive".
```

File: docs/docs/recipes/configure-connectors/social-connector.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+sidebar_label: Social Connector
+sidebar_position: 2
+---
+
+# Social Connector
+
+Suppose you are now on the "Connectors" page in Admin Console. Go to the "Social connectors" tab and add a new connector to enter the configuration process.
+
+:::tip
+You may find that no secondary choice of **_platform_** is required for some connectors such as Google, Facebook, and GitHub. These connectors' **_platform_** are always **_Universal_**. Explore **_platform_** description (TODO: add a link here after connector reference merged) to know more.
```
Comment (gao-sun) [inline docs/docs/recipes/configure-connectors/social-connector.md:11]: generally we don't need italic and bold at the same time.
```suggestion
You may find that some connectors do not have a secondary choice of platform, such as Google, Facebook, and GitHub. These connectors are _Universal connectors_. Explore platform description (TODO: add a link here after connector reference merged) to know more.
```

File: docs/docs/recipes/configure-connectors/README.md [hunk 11]
```diff
@@ -3,3 +3,27 @@ sidebar_position: 5
...
+
+:ghost: **Welcome to the world of _connectors_** :ghost:
+
+_Connectors_ can be classified into three types: _SMS Connector_, _Email Connector_ and _Social Connector_. See [connector reference](../../references/connectors/README.md) for more details about connector type.
+
+You can continue setting up _Email connector_ or _SMS connector_, and you need to go to "Social connectors" tab before adding _Social connector_.
+
+:::note
+Move on to detailed guides for different connector types' configuration: [_SMS or Email connector_](./sms-or-email-connector.md) or [_Social connector_](./social-connector.md).
+:::
```
Comment (gao-sun) [inline docs/docs/recipes/configure-connectors/README.md:29]: optional: I think these two pages are too short. maybe just combine them in 1 page?

**Review by gao-sun — APPROVED** [review]

**Pull request merged**